phase1/gce/configure-vm.sh

```diff
@@ -4,12 +4,14 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-ROLE=$(curl \
-  -H "Metadata-Flavor: Google" \
-  "metadata/computeMetadata/v1/instance/attributes/k8s-role")
-BUCKET=$(curl \
-  -H "Metadata-Flavor: Google" \
-  "metadata/computeMetadata/v1/instance/attributes/k8s-deploy-bucket")
+get_metadata() {
```

Comment:
We need to etch this out when we figure out the contract. This requires a switch on which phase 1 you are using (different impl per cloud provider) so it's convienient to roll it into the phase1 handoff. This make phase2 a bit more genenral.